### DISCUSSION TITLE
Github-Readme-Stats does not show my issues

### DISCUSSION THREAD

**Discussion by LidiaGarrigo** [anuraghazra/github-readme-stats#1906] [Q&A] [answered]
**Github-Readme-Stats does not show my issues.**

When I add the GitHub stats card to my personal repository, I see that it doesn't show me the actual total of issues I have.
In my day to day, I work with issues that I link to projects and the card does not show them.

![Captura de Pantalla 2022-07-30 a las 9 23 00](https://user-images.githubusercontent.com/72071118/181879775-1e22cee8-5efb-42ab-96b4-5668046f4932.png)

Attached is the link I use: `[![Lidia's GitHub stats](https://github-readme-stats.vercel.app/api?username=lidiagarrigo&theme=highcontrast&show_icons=true&count_private=true)](https://github.com/lidiagarrigo/LidiaGarrigo)`

As you can see I have 21 closed and 11 open issues:

![Captura de Pantalla 2022-07-30 a las 9 31 17](https://user-images.githubusercontent.com/72071118/181880108-ac1f532b-263d-4b57-b59a-ed8162285484.png)

I would appreciate help with this problem.

I look forward to hearing from you.
Thank so much.

**Comment by rickstaa** [comment, marked as answer]
Hey @LidiaGarrigo, thanks for your question. I checked your card, and it shows the right amount of issues. I think the confusion is that it is not clear that we only count problems created by your user account (see https://github.com/issues?q=is%3Aopen+is%3Aissue+author%3ALidiaGarrigo+archived%3Afalse+).

https://github.com/anuraghazra/github-readme-stats/blob/86f0ea071595df748a8db294db38d9cb18bd1a47/src/fetchers/stats-fetcher.js#L136

 Feel free to create a pull request to improve the README.md. 🚀 You can see that the card is correct by add ingthe `author:LidiaGarrigo` filter in GitHub's search tool. If you do that, you will only get https://github.com/anuraghazra/github-readme-stats/issues/1904 issue. You can also test this out by running the following code in the GraphQL explorer:

```graphql
query userInfo($login: String!) {
  user(login: $login) {
    openIssues: issues(states: OPEN) {
      totalCount
    }
    closedIssues: issues(states: CLOSED) {
      totalCount
    }
  }
}
```

```graphql
{
  "login": "LidiaGarrigo"
}
```

**Reply by rickstaa** [reply]
@LidiaGarrigo Did that answer your questions?

**Reply by LidiaGarrigo** [reply]
Sorry, I have just read this message.
Yes, thank you so much for solving my doubts.
Have a nice day.

**Reply by rickstaa** [reply]
Great to hear! Could you maybe accept my answer so that we can mark this question as answered 🤓.

**Reply by LidiaGarrigo** [reply]
Of course! ☺️
